faker/providers/file/__init__.py

```diff
@@ -262,6 +274,7 @@ def file_path(
         category: Optional[str] = None,
         extension: Optional[Union[str, Sequence[str]]] = None,
         absolute: Optional[bool] = True,
+        file_system_rule: Optional[str] = "linux",
```

Comment:
Can we specify this as `Literal`?

```suggestion
        file_system_rule: Literal["linux", "windows"] = "linux",
```

Reply:
yes I think this is better type hinting